engine/flink/management/periodic/src/main/scala/pl/touk/nussknacker/engine/management/periodic/PeriodicProperty.scala

```diff
@@ -25,6 +31,9 @@ object PeriodicProperty {
   def nextRunAt(clock: Clock): Either[String, Option[LocalDateTime]]
 }
 
+@JsonCodec case class ComplexPeriodicProperty(schedules: Map[String, PeriodicProperty]) extends BasePeriodicProperty
```

Comment:
Can it be `MultipleSchedulesProperty`? I can't see any complexity here ;)

Reply:
done